s2/edge_distances_test.go

```diff
@@ -202,26 +202,26 @@ func TestEdgeDistancesUpdateMinInteriorDistanceRejectionTestIsConservative(t *te
 			a:       Point{r3.Vector{1, -8.9031850507928352e-11, 0}},
 			b:       Point{r3.Vector{-0.99999999999996347, 2.7030110029169596e-07, 1.555092348806121e-99}},
 			minDist: minDist,
-			want:    false,
+			want:    true,
```

Comment:
Confirmed `true` agrees with C++.

https://github.com/google/s2geometry/blob/58de4ea1e2f8a294e0c072c602c22232fd1433ad/src/s2/s2edge_distances_test.cc#L204

@rsned Maybe we should just delete the `want` field, since it's always `true`.

Reply:
We should probably come up with At least one test case that should be false just to be sure things actually do as they claim.